### PR TITLE
Update `serializers.py` to address #1

### PR DESCRIPTION
**Pull request by rp-20230816**
Addresses the #1 issue.

### CODE REVIEW

**Comment by rp-20230816** [comment]
@gitco-demo-test This is my proposed changes.

File: contacts/serializers.py
Comment (gitco-demo-test) [inline contacts/serializers.py:1]: Just looking at the changes first...

File: contacts/serializers.py [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 # create a ContactSerializer class
 class ContactSerializer(serializers.HyperlinkedModelSerializer):
-    url = serializers.HyperlinkedIdentityField(view_name='contacts:detail')
+    address = serializers.HyperlinkedIdentityField(view_name='contacts:detail')
```
Comment (gitco-demo-test) [inline contacts/serializers.py]: @rp-20230816 I see what you mean now.

Comment (rp-20230816) [reply contacts/serializers.py]: 👍

**Pull request merged**